src/QsFmt/Formatter.Tests/Examples.fs

```diff
@@ -78,3 +78,127 @@ let ``Removes extraneous spaces`` =
+                (7 * 1) // Comments too.
+                + 4 / Fun<Bool, Double>();
+            using (q2 = Qubit()) {
+                let s = $"{1 + 2}";
+            }
+        }
+    }
+}""",
+
+    """namespace Foo {
+    operation Bar() : Unit {
+        use (qubits, q) = (Qubit[2], Qubit()) {
+            let x = // Newlines are preserved.
+                (7 * 1) // Comments too.
+                + 4 / Fun<Bool, Double>();
+            use q2 = Qubit() {
+                let s = $"{1 + 2}";
+            }
+        }
+    }
+}"""
+
+[<Example(ExampleKind.Update)>]
+let ``Updates Using Statements`` =
+    """namespace Foo {
+    operation Bar() : Unit {
+        using ((qubits, q) = (Qubit[2], Qubit()));
+        let x = // Newlines are preserved.
+            (7 * 1) // Comments too.
+            + 4 / Fun<Bool, Double>();
+        using (q2 = Qubit());
+        let s = $"{1 + 2}";
+    }
+}""",
+
+    """namespace Foo {
+    operation Bar() : Unit {
+        use (qubits, q) = (Qubit[2], Qubit());
+        let x = // Newlines are preserved.
+            (7 * 1) // Comments too.
+            + 4 / Fun<Bool, Double>();
+        use q2 = Qubit();
+        let s = $"{1 + 2}";
+    }
+}"""
+
+[<Example(ExampleKind.Update)>]
+let ``Updates Borrowing Blocks`` =
+    """namespace Foo {
+    operation Bar() : Unit {
+        borrowing ((qubits, q) = (Qubit[2], Qubit())) {
+            let x = // Newlines are preserved.
+                (7 * 1) // Comments too.
+                + 4 / Fun<Bool, Double>();
+            borrowing (q2 = Qubit()) {
+                let s = $"{1 + 2}";
+            }
+        }
+    }
+}""",
+
+    """namespace Foo {
+    operation Bar() : Unit {
+        borrow (qubits, q) = (Qubit[2], Qubit()) {
+            let x = // Newlines are preserved.
+                (7 * 1) // Comments too.
+                + 4 / Fun<Bool, Double>();
+            borrow q2 = Qubit() {
+                let s = $"{1 + 2}";
+            }
+        }
+    }
+}"""
+
+[<Example(ExampleKind.Update)>]
+let ``Updates Borrowing Statements`` =
+    """namespace Foo {
+    operation Bar() : Unit {
+        borrowing ((qubits, q) = (Qubit[2], Qubit()));
+        let x = // Newlines are preserved.
+            (7 * 1) // Comments too.
+            + 4 / Fun<Bool, Double>();
+        borrowing (q2 = Qubit());
+        let s = $"{1 + 2}";
+    }
+}""",
+
+    """namespace Foo {
+    operation Bar() : Unit {
+        borrow (qubits, q) = (Qubit[2], Qubit());
+        let x = // Newlines are preserved.
+            (7 * 1) // Comments too.
+            + 4 / Fun<Bool, Double>();
+        borrow q2 = Qubit();
+        let s = $"{1 + 2}";
+    }
+}"""
+
+[<Example(ExampleKind.Update)>]
+let ``Updates Using with Comments`` =
+    """namespace Foo {
+    operation Bar() : Unit {
+        using (
+            q = Qubit() // comment
+        ) {
+        }
+    }
+}""",
+
+    """namespace Foo {
+    operation Bar() : Unit {
+        use 
```

Comment:
This line correctly has a trailing space, however Fantomas wants to remove that and won't allow trailing spaces. I don't want to have to exempt this whole file from Fantomas. Is there something I can do to have Fantomas ignore this trailing space on this line?